.github/workflows/publish-noir-js.yml

```diff
@@ -34,6 +38,9 @@ jobs:
       - name: Build noirc_abi
         run: yarn workspace @noir-lang/noirc_abi build
 
+      - name: Build acvm_js
+        run: yarn workspace @noir-lang/acvm_js build
+
       - name: Build noir_js
         run: yarn workspace @noir-lang/noir_js build
```

Comment:
Noir_js needs the types and backend packages in order to be built. Are you sure that this works?

Reply:
This seems to be working as per this dry run: https://github.com/noir-lang/noir/actions/runs/6376154044/job/17302810816

Reply:
This can sometimes happen due to caching